velox/expression/tests/ArrayViewTest.cpp

```diff
@@ -589,6 +589,8 @@ TEST_F(NullableArrayViewTest, materializeArrayOfCustomTypes) {
   std::vector<std::optional<UDT>> expected = {
       UDT{0}, std::nullopt, UDT{1}, UDT{2}};
   ASSERT_EQ(array, expected);
+
+  UDTTypeRegistrar::unregisterType();
```

Comment:
Thanks. Let's extract these changes into a separate PR. Let's also make sure that this code is called even if test fails. Maybe use folly::makeGuard from folly/ScopeGuard.h